allennlp/service/servable/models/simple_tagger.py

```diff
@@ -0,0 +1,61 @@
+import codecs
+import os
+from tempfile import gettempdir
+
+from allennlp.common import Params
+from allennlp.data.dataset_readers.sequence_tagging import SequenceTaggingDatasetReader
+from allennlp.data.fields import TextField
+from allennlp.data.token_indexers import SingleIdTokenIndexer
+from allennlp.data.tokenizers import WordTokenizer
+from allennlp.data.vocabulary import Vocabulary
+from allennlp.models.simple_tagger import SimpleTagger
+from allennlp.service.servable import Servable, JSONDict
+
+# Simple Tagger Model
+class SimpleTaggerServable(Servable):
```

Comment:
I thought about that, I'm just not sure that `Model` is the exact right interface for this. Its only method returns a `Dict[str, torch.Tensor]`, that's a bit of a leaky abstraction, and I'm not sure if it covers every one of our use cases (e.g. we might want our web demo to also return a vocabulary or some strings or something?)